gokce_fingerprint_verification.py

```diff
@@ -0,0 +1,184 @@
+######################################################################
+# Names of any others you worked with:
+# AI transcript if used: Claude at moments I got really stuck.
+# Any extensions done:
+# Name: Gokce Can Ertem
+######################################################################
+
+# Function to read the file and turn it into a fingerprint
+def read_and_list_fprint(filename):
+    """Reads the given print and returns a dictionary with all info."""
+    # Your turn!
+    
+
+
+    with open (filename) as fprint:
+        #lines = fprint.read().strip().splitlines()
+        lines = fprint.read().splitlines()
+        fprint2dlist = [list(line) for line in lines]
```

Comment:
This just broke up the entire file into a list of lists of individual characters. Which would have been fine for purely the fingerprint portion, but you needed to parse out the other pieces and save everything in a dictionary, not a list.